accord-maelstrom/build.gradle

```diff
@@ -20,8 +20,13 @@ plugins {
     id 'accord.java-conventions'
 }
 
+compileJava {
```

Comment:
can you move the 1.8 logic to buildSrc/src/main/groovy/accord.java-conventions.gradle